floor.go

```diff
@@ -104,6 +114,24 @@ func (f *Floor) watchFloorPrice() {
 					time.Sleep(time.Duration(f.Frequency) * time.Second)
 				}
 
+				// Custom activity messages
+				if len(custom_activity) > 0 {
+
+					// Display the real activity once per cycle
+					if itr == len(custom_activity) {
+						itr = 0
+						itrSeed = 0.0
+						f.Activity = activity
+					} else if math.Mod(itrSeed, 2.0) == 1.0 {
+						f.Activity = custom_activity[itr]
+						itr++
+						itrSeed++
+					} else {
+						f.Activity = custom_activity[itr]
+						itrSeed++
+					}
+				}
+
 				err = dg.UpdateGameStatus(0, activity)
```

Comment:
```suggestion
				err = dg.UpdateGameStatus(0, f.Activity)
```

Also need this same change on line 139 but I cant make a suggestion there so youll need to do it.

This actually uses the custom activity :)

Reply:
Good one!